.github/actions/next-integration-stat/src/index.ts

```diff
@@ -348,158 +234,16 @@ async function getJobResults(
 
 // Get the latest base test results to diff against with current test results.
 async function getTestResultDiffBase(
-  octokit: Octokit,
-  shouldDiffWithMain: boolean
+  _octokit: Octokit
 ): Promise<TestResultManifest | null> {
-  console.log('Trying to find latest test results to compare')
-
-  // First, get the tree of `test-results` from `nextjs-integration-test-data` branch
```

Comment:
This code is only used for the comment-on-PR codepath that's currently dead.

The git branch this is referring to (https://github.com/vercel/turborepo/tree/nextjs-integration-test-data/test-results) hasn't been updated in >2 years. Any data there is irrelevant, and this should probably be engineered another way (committing it to git, even in a different branch, adds a ton of bloat to the repo).